.github/pull_request_template.md

```diff
@@ -1,8 +1,23 @@
-<!-- Thank for your PR! If it's your first time contributing to Taichi, please make sure you have read Contributor Guideline(https://taichi.readthedocs.io/en/latest/contributor_guide.html) (last update: March 26, 2019). -->
+<!--
+Thank for your PR!
+If it's your first time contributing to Taichi, please make sure you have read our Contributor Guideline:
+  https://taichi.readthedocs.io/en/latest/contributor_guide.html
 
-<!-- Please always prepend your PR title with tags such as [Metal], [CUDA], [Doc], [Example]. Use a lowercased tag (e.g. [cuda]), for PRs that are invisible to end-users (e.g. intermediate implementation). More details: http://taichi.readthedocs.io/en/latest/contributor_guide.html#prtags -->
+- Please always prepend your PR title with tags such as [CUDA], [Lang], [Doc], [Example], e.g.:
+    [Lang] Add ti.Complex as Taichi class
+- Use a lowercased tag for PRs that are invisible to end-users, i.e., won't be highlighted in changelog:
+    [cuda] [test] Fix out-of-memory error while running test
+- More details: http://taichi.readthedocs.io/en/latest/contributor_guide.html#prtags
 
-Related issue = #... (if any)
+- Please fill the following blank with the issue number this PR related to (if any):
+    Related issue = #2345
+- If your PR will fix the issue **completely**, use the `close` keyword:
+    Related issue = close #2345
+- So that when the PR get's merged, GitHub will **automatically** close the issue #2345 for you :)
+- If the PR doesn't belong to any existing issue, feel free to leave it blank :)
+  -->
+Related issue = #
```

Comment:
Suggestion: I noticed that I made a dump thing in #1189 by capitalizing two tags. This resulted in v0.6.9's log to print my PR in two sections, see https://github.com/taichi-dev/taichi/pull/1215#issue-432631674.. WDYT if we suggest to always capitalize one tag, whichever is more relevant to the PR?